pdf_build_src/process_markdowns.py

```diff
@@ -141,6 +141,164 @@ def modify_changelog():
         file.writelines(data)
 
 
+"""Number of chars maximal in one line approximated from a line of the PDF"""
```

Comment:
no multiline comment syntax needed for a single line comment

also, will we need to adjust this in the future? was it "visually" determined? can we instead auto determine it?

```suggestion
# Number of chars maximal in one line approximated from a line of the PDF
```